keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExceptionHandler.kt

```diff
@@ -88,10 +89,10 @@ class ExceptionHandler(
   @ResponseStatus(FORBIDDEN)
   fun onAccessDenied(e: AccessDeniedException): ApiError {
     log.error(e.message)
-    val authFailure = FiatPermissionEvaluator.getAuthorizationFailure().get()
+    val authFailure: AuthorizationFailure? = FiatPermissionEvaluator.getAuthorizationFailure().orElse(null)
```

Comment:
This was also throwing exceptions. It didn't seem to break any tests directly but it was certainly incorrect.